<ui version="4.0" >
 <class>dEditWebForm</class>
 <widget class="Q3MainWindow" name="dEditWebForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Web Form</string>
  </property>
  <widget class="QWidget" name="widget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>445</width>
     <height>328</height>
    </rect>
   </property>
   <layout class="QGridLayout" >
    <item row="0" column="0" >
     <widget class="QTabWidget" name="tabWidget10" >
      <widget class="QWidget" name="tab" >
       <attribute name="title" >
        <string>Common</string>
       </attribute>
       <layout class="QGridLayout" >
        <item row="1" column="0" >
         <widget class="QLabel" name="textLabel2" >
          <property name="text" >
           <string>Description</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0" >
         <widget class="Q3TextEdit" name="eDescription" />
        </item>
        <item row="0" column="0" >
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QLabel" name="textLabel1" >
            <property name="text" >
             <string>Name:</string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="eName" />
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="TabPage" >
       <attribute name="title" >
        <string>Form editor</string>
       </attribute>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="Q3TextEdit" name="eFormSource" >
          <property name="textFormat" >
           <enum>Qt::PlainText</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab" >
       <attribute name="title" >
        <string>Client module</string>
       </attribute>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="QSEditor" native="1" name="eClientModule" />
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="TabPage" >
       <attribute name="title" >
        <string>Server module</string>
       </attribute>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="QSEditor" native="1" name="eServerModule" />
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="TabPage" >
       <attribute name="title" >
        <string>Aliases</string>
       </attribute>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="Q3Table" name="tAliases" >
          <property name="numRows" >
           <number>0</number>
          </property>
          <property name="numCols" >
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="5" margin="5" />
 <pixmapfunction>rcIcon</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3MainWindow</class>
   <extends>QWidget</extends>
   <header>q3mainwindow.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3Table</class>
   <extends>Q3Frame</extends>
   <header>q3table.h</header>
  </customwidget>
  <customwidget>
   <class>Q3TextEdit</class>
   <extends>Q3Frame</extends>
   <header>q3textedit.h</header>
  </customwidget>
  <customwidget>
   <class>QSEditor</class>
   <extends>QWidget</extends>
   <header>qseditor.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local" >aliaseditor.h</include>
  <include location="local" >mdtree.h</include>
  <include location="local" >mainform.h</include>
  <include location="local" >qseditor.h</include>
  <include location="local" >qseditor.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
